horizons/ai/aiplayer/productionbuilder.py

```diff
@@ -263,24 +263,24 @@ def display(self):
 			return
 
 		tile_colors = {
-			BUILDING_PURPOSE.ROAD:            ( 30,  30,  30),
```

Comment:
Yes, I find it look very nice with the table layout, too.
But this break some code style rules. And it could be color, but I am not sure.
But if it a color maybe it is possible make a hexadecimal color definition or real names?
I think hexadecimal is one step better, real names colors two steps.

https://de.wikipedia.org/wiki/Hexadezimale_Farbdefinition